spotify

```diff
@@ -26,6 +26,35 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+USER_CONFIG_DEFAULTS='CLIENT_ID=""
+CLIENT_SECRET=""'
+USER_CONFIG_FILE="${HOME}/.shpotify.cfg"
+if ! [[ -f "${USER_CONFIG_FILE}" ]]; then
+    touch "${USER_CONFIG_FILE}"
+    echo "${USER_CONFIG_DEFAULTS}" > "${USER_CONFIG_FILE}"
+fi
+source "${USER_CONFIG_FILE}"
```

Comment:
Just put these at the top so users can't overwrite any existing variables -- happy do something else if you have any better ideas.

Also happy to change the variable names if you want the user-interface to be any different